packages/website/assets/icons/infoA.js

```diff
@@ -0,0 +1,23 @@
+import * as React from 'react';
+
+const InfoAIcon = props => (
+  <svg xmlns="http://www.w3.org/2000/svg" width={9.038} height={9.184} viewBox="0 0 9.038 9.184" {...props}>
+    <g opacity={0.75}>
+      <path d="M4.519 0A4.519 4.519 0 1 1 0 4.519 4.519 4.519 0 0 1 4.519 0Z" fill="#fff" />
+      <text
+        transform="translate(3.019 6.184)"
+        fill="#131721"
+        stroke="rgba(0,0,0,0)"
+        strokeWidth={2}
+        fontSize={5}
+        fontFamily="Helvetica"
```

Comment:
nit: what happpens here if the users system doesn't have Helvetica? Do svgs fallback in the same way as htmls? what if the users default font is butt-ugly, or has awkward glyph positioning relative to the containgin element? Might be better to convert a question mark to an svg path.